### DISCUSSION TITLE
如何在事件过滤里嵌套一个事件过滤，类似if嵌套

### DISCUSSION THREAD

**Discussion by ConstantineQAQ** [ForteScarlet/simpler-robot#416] [不要在这里发帖子了]
如何在事件过滤里嵌套一个事件过滤，类似if嵌套
我想写一个俄罗斯轮盘，发送：“开启轮盘”开启游戏，然后在这个事件函数里发送“开枪”，随机禁言

**Comment by ForteScarlet** [comment]
你或许可以考虑使用 [**持续会话**](https://simbot.forte.love/docs/basic/continuous-session)，但是目前持续会话的外部文档尚不完善，如要使用请根据代码及注释摸索，且可能存在隐患（处于实验阶段的API）

如果不使用持续会话，或许考虑将两个逻辑拆分为分别的监听函数，并在监听函数体外部提供“状态”，来处理两个事件的流转。

或者其他方式，能实现就行。
